src/main/java/com/google/sps/data/Comment.java

```diff
@@ -26,6 +33,21 @@ public long getID() {
         return id; 
     }
 
+    public static boolean isCommentToxic(String msg) {
```

Comment:
public static methods should go above the constructor

go/java-practices/class-structure.md